integration_tests/mockApis/educationAndWorkPlanApi.ts

```diff
@@ -323,9 +323,36 @@ const stubGetInduction500Error = (prisonNumber = 'G6115VJ'): SuperAgentRequest =
     },
   })
 
+const stubCreateInduction = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'POST',
+      urlPattern: `/inductions/${prisonNumber}`,
+    },
+    response: {
+      status: 201,
+    },
+  })
+
+const stubRedirectToPlpAfterCreateInduction = (prisonNumber = 'G6115VJ'): SuperAgentRequest =>
+  stubFor({
+    request: {
+      method: 'GET',
+      urlPattern: `/plan/${prisonNumber}/induction-created`,
```

Comment:
I don't understand this part. Or rather, it's not the educationAndWorkPlanApi that's being mocked here is it? 🤯

Reply:
See this in the PLP codebase, that might explain why:
https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/blob/main/server/routes/postInductionCreation/index.ts

Reply:
Ah yes, I forgot about the need to hand back to the PLP UI - that makes complete sense, thank you 👍